categorize_adjectives/libraries/bert.py

```diff
@@ -0,0 +1,169 @@
+"""Contains function for calculating BERT similarity"""
```

Comment:
Can you add a link to this source code?

Reply:
This was written by one of my colleague at ETH